File: libs/python/test-one.py
import mf2py
import sys
import json

if len(sys.argv) < 3:
    print('usage: '+ sys.argv[0]+ ' <input_file> <base_url>')
    sys.exit(1)

name = sys.argv[1]
base = sys.argv[2]

with open(name,'r') as file:
    p = mf2py.Parser(doc=file, url=base)
    res = json.loads(p.to_json())
    del res['debug']
    print(json.dumps(res))

File: libs/js/test-one.js
const { mf2 } = require('microformats-parser');

var infile = process.argv[2];
var base = process.argv[3];

if (infile === undefined || base === undefined) {
  console.log("Usage: " + process.argv[1] + " <input_file> <base_url>");
  process.exit(1);
}

var fs = require("fs");

fs.readFile(infile, "utf8", function (err, data) {
    if (err) throw err;

    const parsed = mf2(data, {
        baseUrl: base,
    });
    console.log("%j", parsed);
});
